src/styles/blocks/form-field--small.css

```diff
@@ -0,0 +1,26 @@
+.form-field__second {
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  background-image: url(/src/images/Search.svg);
+  background-size: 11px 11px;
+  background-repeat: no-repeat;
+  background-position: 13px center;
+  padding-left: 33px;
+  height: 41px;
+  width: 100vw;
+  padding-block: 0;
+}
+
+.form-field__second:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.form-field__second:focus {
+  box-shadow: 0;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: bold;
```

Comment:
Ensure that the 'Avenir' font family includes a bold weight. If not, the `font-weight: bold` style might not render as expected. Consider specifying a font-weight that is available in your font files.